setup.py

```diff
@@ -4,7 +4,7 @@
 requirements = [
     'jupyter==1.0.0',
     'numpy==1.21.5',
-    'matplotlib==3.4',
+    'matplotlib==3.5.1',
```

Comment:
Can you add `matplotlib_inline`?

Reply:
As mentioned in https://github.com/d2l-ai/d2l-en/pull/2078#discussion_r832815722 `matplotlib_inline` is an ipython/ipykernel dependency which is automatically installed when jupyter is installed since `jupyter` requires `ipykernel` and `ipykernel` ultimately requires `matplotlib_inline`. If we pin it here, it might cause conflicts with the jupyter version pinned.